expertsystem/data.py

```diff
@@ -126,3 +129,36 @@ def keys(self) -> KeysView[str]:
 
     def values(self) -> ValuesView[Particle]:
         return self.__particles.values()
+
+
+class QuantumState(NamedTuple):
+    """Collection of properties defining a quantum mechanical state.
+
+    Related to `.Particle`, but can contain more specific quantum state
+    information, such as `.Spin` projection.
+    """
+
+    charge: int
+    spin: Spin
+    mass: float
+    strangeness: int = 0
+    charmness: int = 0
+    bottomness: int = 0
+    topness: int = 0
+    baryon_number: int = 0
+    electron_number: int = 0
+    muon_number: int = 0
+    tau_number: int = 0
+    width: Optional[float] = None
```

Comment:
Remove. Same reasoning as for mass

Reply:
Problem is: the `Particle` class is the starting point, which should contain info from the PDG (where it is called mass and width). For me `ComplexEnergyState` is the mathematical generalisation from that (for internal use and for the output recipe that is used for PWA).